WireDomain/Sources/WireDomain/Synchronization/SyncManager.swift

```diff
@@ -250,4 +294,24 @@ final class SyncManager: SyncManagerProtocol {
         }
     }
 
+    private func logSyncTime(
+        for phase: SyncPhaseName,
+        completedAllPhases: Bool = false
+    ) {
+        let syncType = phase != .pullPendingEvents ? "slow sync" : "quick sync"
+        let currentTime = Date.now
+        let duration = currentTime.timeIntervalSince(syncTimeTracker.phaseStartTime)
+        let message = "Completed \(syncType) phase \(phase) in \(duration)"
+        WireLogger.sync.info(message, attributes: .safePublic)
```

Comment:
```suggestion
        let message = "SyncPhase completed"
        WireLogger.sync.info(message, attributes: [.duration: \(duration), .syncType: syncType, .phase: phase], .safePublic)
```
better to move the info inside attributes so in datadog we can extract it directly.

Reply:
done in https://github.com/wireapp/wire-ios/pull/2373/commits/aaa66388f4f307e3057b6f1dc3a4fe2d7a18c582